### PR TITLE
media-sound/spotify-tui: +

### PR DESCRIPTION
**Pull request by BGazotti**
r/SuddenlyCaralho?

A nice spotify terminal client. Works great with spotifyd - might add an ebuild for that one soon. Glad to be of service to the Gentoo community!

### CODE REVIEW

File: media-sound/spotify-tui/spotify-tui-0.24.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+src_unpack() {
+	cargo_src_unpack
+}
+
+src_compile() {
+	#cargo_src_compile
+	cargo build --release --target-dir "target/"
+}
+
```
Comment (leonardohn) [inline media-sound/spotify-tui/spotify-tui-0.24.0.ebuild:299]: This part can be omitted, as it is the default behavior on cargo.eclass

File: media-sound/spotify-tui/spotify-tui-0.24.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+	# install documentation
+	dodoc "${S}"/README.md
+	# install spt binary
+	dobin "${S}"/target/release/spt
+}
+
+pkg_postinst() {
+	einfo "Cleaning up the rust..."
+	rm -r "${S}"/target/release/build
+	rm -r "${S}"/target/release/deps
```
Comment (leonardohn) [inline media-sound/spotify-tui/spotify-tui-0.24.0.ebuild:311]: This part can also be omitted

File: media-sound/spotify-tui/spotify-tui-0.24.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+	#cargo_src_compile
+	cargo build --release --target-dir "target/"
+}
+
+src_install() {
+
+	# install documentation
+	dodoc "${S}"/README.md
+	# install spt binary
+	dobin "${S}"/target/release/spt
```
Comment (leonardohn) [inline media-sound/spotify-tui/spotify-tui-0.24.0.ebuild:305]: It is better to use `cargo_src_install` here

File: media-sound/spotify-tui/spotify-tui-0.24.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,314 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CRATES="addr2line-0.14.1
+		adler-0.2.3
+		adler32-1.2.0
```
Comment (leonardohn) [inline media-sound/spotify-tui/spotify-tui-0.24.0.ebuild:8]: Either add indentation to everything or leave it unindented

**Review by leonardohn — CHANGES_REQUESTED** [review]
Thanks for the contribution!

I did a few suggestions in the ebuild, please take a look.

**Comment by BGazotti** [comment]
Very well, I'll take a look at it. I'm fairly new to this ebuild writing business, so I apologize for any obvious mistakes. Should be re-committing soon!

**Comment by leonardohn** [comment]
Thanks!

**Pull request merged**